### ISSUE TITLE
Rename Home screen to Home app? 

### ISSUE THREAD

**Issue by mj-sson** [zobnin/aio#97]
This might be hair-splitting, but "Home screen" doesn't sound quite right to me when talking about selecting the default Launcher app. 

I think you would normally use the term "Home app" in this context:

![Screenshot_20231119-211646~2.png](https://github.com/zobnin/aio/assets/121413314/4c8e80ef-30ae-46e1-8f86-a5c68566e27f)

![Screenshot_20231119-212114~2.png](https://github.com/zobnin/aio/assets/121413314/be978360-b5b6-48d6-883e-a8e2b7d808c2)

**Comment by zobnin** [comment]
This name may vary from phone to phone, so on some phones, it might be called the home screen or launcher.

**Comment by mj-sson** [comment]
Yes, the term "Home screen" is sometimes used in place of the more common and obvious terms "Launcher" and "Home app".

However this term is too vague and ambiguous to be used without further clarification:

To me as a user, the term "Home SCREEN" is mostly used to describe the screen you see, when you press the Home Button. Not the actual "Home app/Launcher" itself.

This makes "Select home screen" sound like you are selecting a "Home page/screen" on a regular Launcher. 

I think it would be clearer if you simply replaced the term "Home screen" with either "Home screen app", "Home app" or simply with "Launcher". I would also prepend "default" to the term. 


**TL;DR**
 
OLD: "Select home screen" >
**NEW: "Select default home screen app"**

**Comment by mj-sson** [comment]
**TL;DR**
 
OLD: "Select home screen" 
**NEW: "Select default home screen app"**

**Comment by mj-sson** [comment]
I think it would be clearer if you just replaced the term "Home screen" in the Basic settings, with either "Home screen app", "Home app" or with "Launcher". I would also prepend "default" to the term.

TL;DR

OLD: "Select home screen"
NEW: "Select default home screen app"

![Screenshot_20231119-211646.png](https://github.com/zobnin/aio/assets/121413314/a25c13bb-7c2b-4a4e-ba6d-ff2da8990e5a)
